webpack/assets/javascripts/react_app/common/hooks/API/APIHooks.test.js

```diff
@@ -34,3 +34,15 @@ it('shuold use the given key', async () => {
   await waitForNextUpdate();
   expect(result.current.key).toEqual(API_TEST_KEY);
 });
+
+it('shuold use the given key', async () => {
```

Comment:
s/shuold/should/

Reply:
This test should have a different name as there is already a test named `'shuold use the given key'`. Would be nice to fix the spelling on the previous test :)